style.css

```diff
@@ -0,0 +1,201 @@
+html {
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+}
+
+body {
+  background-color: #eceef1;
+  display: grid;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav {
+  display: grid;
+  justify-content: end;
+  text-align: right;
+}
+
+input {
+  text-align: left;
+  height: 34px;
+  width: 95%;
+  min-width: 160px;
+  border-radius: 4px;
+  padding-left: 10px;
+  color: #000;
+  margin-top: -200px;
+  font-size: 1.2em;
+}
+
+input:focus {
+  border-color: blue;
+}
+
+footer {
+  position: absolute;
+  text-align: center;
+  text-decoration: none;
+  color: #505050;
+  margin-top: 59rem;
+  margin-left: 32rem;
+}
```

Comment:
Add a margin-bottom to your footer. Take a look at the original page and see how they give some space after the text to let it "breathe" in visual terms.

Check the font-size: yours is larger than the original page.